### ISSUE TITLE
Need a script to restart a vHive cluster (single node)

### ISSUE THREAD

**Issue by ustiugov** [ease-lab/vhive#151]
We need a script that can clean and restart a cluster in one push button.

Existing solution:
```
./scripts/github_runner/clean_cri_runner.sh
sudo containerd 1>~/ctrd-logs/ctrd.out 2>~/ctrd-logs/ctrd.err &
sudo /usr/local/bin/firecracker-containerd --config /etc/firecracker-containerd/config.toml 1>~/ctrd-logs/fccd.out 2>~/ctrd-logs/fccd.err &
source /etc/profile && cd ~/vhive && go build && sudo ./vhive 1>~/ctrd-logs/orch.out 2>~/ctrd-logs/orch.err &
~/vhive/scripts/cluster/create_one_node_cluster.sh
```

There are errors when trying that on CloudLab:
```
../vhive/scripts/github_runner/clean_cri_runner.sh: line 38: ifconfig: command not found
../vhive/scripts/github_runner/clean_cri_runner.sh: line 39: ifconfig: command not found
../vhive/scripts/github_runner/clean_cri_runner.sh: line 40: bridge: command not found
../vhive/scripts/github_runner/clean_cri_runner.sh: line 40: jq: command not found
```

Workaround: 
```
export PATH=$PATH:/sbin/
```

**Comment by ustiugov** [comment]
@shyamjesal any update on this?

**Comment by shyamjesal** [comment]
added in https://github.com/ease-lab/vhive/pull/194

**Comment by ustiugov** [comment]
it's not 194. the question is about restarting a cluster on bare metal not restarting the runners.

**Comment by shyamjesal** [comment]
My bad. I will remove the links.

**Comment by ustiugov** [comment]
fixed by #208
